### PR TITLE
Add desktop css version

### PR DESCRIPTION
**Pull request by wassimchakib**
In this feature : 
  • I changed CSS code to be mobile first
  • I added responsive design to support desktop version ( wider than 768 px)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -60,328 +67,425 @@ body {
...
+.card img.snapshot {
+  border-radius: 8px;
+  align-self: center;
 }
 
-/* content */
+.left-block {
+  display: flex;
+  flex-flow: column;
+  gap: 20px;
```
Comment (ridwanediallo) [inline style.css:136]: - [x] Hey @wassimchakib, you did a great job on this project. 👏 However, to make it look nicer, I would suggest you to increase a bit the padding-left of the left block. 
Like this 👇️
```.left-block {     padding-right: 129px; }```

File: style.css [hunk 2]
```diff
@@ -60,328 +67,425 @@ body {
...
   }
 
-  #language {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    height: 72px;
+  .card .right {
+    order: -1;
   }
```
Comment (ridwanediallo) [inline style.css]: - [x] And here as well, I think you can increase the padding-left, this is going to help your portfolio look more like the Figma template. 🤗

` .card . right { padding-left: 20px; }`

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @wassimchakib , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔
- Your projet looks great. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sajid-munawar — APPROVED** [review]
### Hi @wassimchakib , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
 ## [Optional] suggestions:
 * [ ]  Please check my inline comments.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @sajid-munawar in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**